### PR TITLE
Check RPC build in buildimage-mlnx-all-pr job

### PR DESCRIPTION
**Pull request by qiluo-msft**

### CODE REVIEW

File: jenkins/mellanox/buildimage-mlnx-all-pr/Jenkinsfile [hunk 1]
```diff
@@ -33,7 +33,8 @@ git submodule foreach --recursive '[ -f .git ] && echo "gitdir: $(realpath --rel
 
 make configure PLATFORM=mellanox
 touch target/debs/stretch/*.deb
-make SONIC_CONFIG_BUILD_JOBS=1 all
+make SONIC_CONFIG_BUILD_JOBS=1 ENABLE_SYNCD_RPC=y all
+make SONIC_CONFIG_BUILD_JOBS=1 target/sonic-mellanox.bin
```
Comment (lguohan) [inline jenkins/mellanox/buildimage-mlnx-all-pr/Jenkinsfile]: all should include target/sonic-mellanox.bin?

Comment (qiluo-msft) [reply jenkins/mellanox/buildimage-mlnx-all-pr/Jenkinsfile]: yes

Comment (lguohan) [reply jenkins/mellanox/buildimage-mlnx-all-pr/Jenkinsfile]: then, why do we need line 37?

Comment (qiluo-msft) [reply jenkins/mellanox/buildimage-mlnx-all-pr/Jenkinsfile]: If `ENABLE_SYNCD_RPC=y`, then `libsairedis` and all its downstream binaries will be different, including `target/sonic-mellanox.bin`.
Both lines are useful to make sure building works with different environment variables

Comment (lguohan) [reply jenkins/mellanox/buildimage-mlnx-all-pr/Jenkinsfile]: it still looks strange can we figured out the root cause

**Pull request merged**